app/views/admin/users/index.html.erb

```diff
@@ -1,10 +1,15 @@
-<h1><%= t(".title") %></h1>
-<% @users.each do |user| %>
-  <ul id="user_<%= user.id %>">
-    <li><%= t(".name") %>: <%= user.name %></li>
-    <li><%= t(".email") %>: <%= user.email %></li>
-    <li><%= t(".admin") %>: <%= user.admin? %></li>
-    <li><%= link_to t(".links.delete"), admin_user_path(user), method: :delete %>
-      | <%= link_to t(".links.edit"), edit_admin_user_path(user) %></li>
-  </ul>
-<% end %>
+<table>
+  <h3 class="table_align"><%= t(".title") %></h3>
+  <th><%= t(".name") %></th>
+  <th><%= t(".email") %></th>
+  <th><%= t(".admin") %></th>
```

Comment:
These `<th>` still need to be wrapped inside a `<tr>` and their number should match the number of `<td>` you have in other table rows.

Reply:
Done in sha: a2fa585